rubygem-foreman-tasks/rubygem-foreman-tasks.spec

```diff
@@ -1,19 +1,9 @@
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
-%if 0%{?scl:1}
```

Comment:
All this spec depends on https://github.com/theforeman/foreman-tasks/pull/235 being merged though.